ramls/traits/sortable.raml

```diff
@@ -0,0 +1,9 @@
+queryParameters:
+  sort:
+    displayName: Sort options
+    type: string
+    enum: [name, relevance]
```

Comment:
There is an issue with enum objects used as query parameters: 
enum objects that are generated from raml are generated like this:
```
public enum EholdingsPackagesGetSort {
  NAME("name"),

  RELEVANCE("relevance");

  private String name;

  EholdingsPackagesGetSort(String name) {
    this.name = name;
  }
}
```
RestVerticle then compares them with query parameters using case senstive comparison with value returned from toString(), so query parameters will be parsed correctly only if we send this in query "?sort=RELEVANCE", 
we can use String parameter instead of enum and then compare them manually properly, 
or maybe there is some other way around this issue

Reply:
agree with you, Andrii; violates the module contract that is why I changed the values inside raml files to string values and expanded 'example' property with possible values like in the following image
![image](https://user-images.githubusercontent.com/37537790/47654321-345d3800-db93-11e8-8d04-31442cb58b4b.png)